templates/.env.template

```diff
@@ -0,0 +1,62 @@
+#### Image Names & Prefixes ####
+HAVEGED_IMAGE_PREFIX=gcr.io/hedera-registry/
+NETWORK_NODE_IMAGE_PREFIX=gcr.io/hedera-registry/
+NETWORK_NODE_IMAGE_NAME=main-network-node
+
+#### Image Tags/Hashes ####
+NETWORK_NODE_IMAGE_TAG={{nodeImageTag}}
+HAVEGED_IMAGE_TAG={{havegedImageTag}}
+
+#### Java Process Settings ####
+PLATFORM_JAVA_HEAP_MIN=256m
+PLATFORM_JAVA_HEAP_MAX=2g
+PLATFORM_JAVA_OPTS="-XX:+UnlockExperimentalVMOptions -XX:+UseZGC -Xlog:gc*:gc.log"
+
+#### Bind Mount Settings ####
+NETWORK_NODE_LOGS_ROOT_PATH=./network-logs/node
+APPLICATION_ROOT_PATH=./compose-network/network-node
+APPLICATION_CONFIG_PATH=./compose-network/network-node/data/config
+
+#### Network Node Memory Limits ####
+NETWORK_NODE_MEM_LIMIT=8gb
+
+#### Uploader Image Prefixes & Tags ####
+UPLOADER_IMAGE_PREFIX=gcr.io/hedera-registry/
+UPLOADER_IMAGE_TAG={{uploaderImageTag}}
+PYTHON_VERSION=python3.9
```

Comment:
Should be able to change python version, because newer version of the uploader works only with python 3.7 and python 3.6